### DISCUSSION TITLE
Shortcutkey to Open Control Panel

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#8438] [Ideas]
Back to an [old discussion](https://groups.google.com/g/tiddlywiki/c/zRZRmsOWYA0/m/dOVGbCXlAwAJ), I would recommend having a shortcutkey to open the $:/ControlPanel.

I suggest configurable `alt+c`.

I the idea is accepted, I can create a PR.

**Comment by pmario** [comment]
I use the browser addOn BibitNow which also uses ALT-C -> So what about CTRL-ALT-C as a default. 

FF and Edge do use CTRL and CTRL-SHIFT by default ... So CTRL-ALT-C should not clash with them.

**Comment by kookma** [comment]
Okay! I submit a PR

**Comment by kookma** [comment]
@pmario 
I have created the PR. Please see: https://github.com/TiddlyWiki/TiddlyWiki5/pull/8439
For documentation I assume to create a new PR, or should I add more commits here?

**Comment by kookma** [comment]
The PR addresses this idea:
https://github.com/TiddlyWiki/TiddlyWiki5/pull/8439#issuecomment-2254507342

**Comment by linonetwo** [comment]
Could be done by command palette, which can even directly goto some plugin's config. I can also search control panel with custom addon.
